src/monitors/newdeployment.js

```diff
@@ -0,0 +1,44 @@
+const EventEmitter = require('events');
+const config = require('config');
+const kube = require('../kube');
+
+class DeploymentStatus extends EventEmitter{
+  constructor(){
+    super();
+  }
+
+  start(){
+    setInterval(() => {
+      this.check();
+  }, config.get('interval'));
+
+    return this;
+  }
+
+  async check(){
+    let containers = await kube.getContainerStatuses();
```

Comment:
I think rather than checking for newly deployed containers, we should check for newly deployed pods. This is available in `kube.getPods();`